.storybook/themes/news.scss

```diff
@@ -459,6 +459,40 @@
 				),
 			),
 			blocks: (
+				header-nav-chain: (
+					background-color: var(--global-black),
```

Comment:
this black could be a token for different variants in the theme file. this only varies per site, in the blocks.json. If it needed to be dynamic (different color theme per page of site), then it might need different direction

Reply:
@JackHowa Its was determined to not have multiple color configurations per site (like dark / light mode)

Reply:
@JackHowa It was discussed to forgo multiple themes on a per site basis including dark/light mode.